### PR TITLE
Fix windows installation

### LINKED ISSUE #4947: won't able to bring up the project because of libmagic related error
### What I'm trying to achieve
Bring up the project

### Steps to reproduce the problem
1. 
2. 

### What I expected to happen
I can bring up the project

### Screenshots

**System information**
Operating system: Mac osx 
Browser: chrome

This the output error when I run the `python manage.py runserver` or `python manage.py migrate` command 
But I already have libmagic install by running `pip install libmagic`

```
INFO django.utils.autoreload Watching for file changes with StatReloader [PID:75408:MainThread]
INFO django.utils.autoreload Watching for file changes with StatReloader [PID:75408:MainThread]
Exception in thread django-main-thread:
Traceback (most recent call last):
  File "/Users/yy/Documents/software/brew/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/threading.py", line 917, in _bootstrap_inner
    self.run()
  File "/Users/yy/Documents/software/brew/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/threading.py", line 865, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/yy/Documents/POC/saleor/lib/python3.7/site-packages/django/utils/autoreload.py", line 54, in wrapper
    fn(*args, **kwargs)
  File "/Users/yy/Documents/POC/saleor/lib/python3.7/site-packages/django/core/management/commands/runserver.py", line 109, in inner_run
    autoreload.raise_last_exception()
  File "/Users/yy/Documents/POC/saleor/lib/python3.7/site-packages/django/utils/autoreload.py", line 77, in raise_last_exception
    raise _exception[1]
  File "/Users/yy/Documents/POC/saleor/lib/python3.7/site-packages/django/core/management/__init__.py", line 337, in execute
    autoreload.check_errors(django.setup)()
  File "/Users/yy/Documents/POC/saleor/lib/python3.7/site-packages/django/utils/autoreload.py", line 54, in wrapper
    fn(*args, **kwargs)
  File "/Users/yy/Documents/POC/saleor/lib/python3.7/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/Users/yy/Documents/POC/saleor/lib/python3.7/site-packages/django/apps/registry.py", line 114, in populate
    app_config.import_models()
  File "/Users/yy/Documents/POC/saleor/lib/python3.7/site-packages/django/apps/config.py", line 211, in import_models
    self.models_module = import_module(models_module_name)
  File "/Users/yy/Documents/POC/saleor/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/yy/Documents/POC/saleor/saleor/saleor/extensions/models.py", line 5, in <module>
    from saleor.core.utils.json_serializer import CustomJsonEncoder
  File "/Users/yy/Documents/POC/saleor/saleor/saleor/core/utils/__init__.py", line 22, in <module>
    from versatileimagefield.image_warmer import VersatileImageFieldWarmer
  File "/Users/yy/Documents/POC/saleor/lib/python3.7/site-packages/versatileimagefield/image_warmer.py", line 11, in <module>
    from .utils import (
  File "/Users/yy/Documents/POC/saleor/lib/python3.7/site-packages/versatileimagefield/utils.py", line 7, in <module>
    import magic
  File "/Users/yy/Documents/POC/saleor/lib/python3.7/site-packages/magic.py", line 181, in <module>
    raise ImportError('failed to find libmagic.  Check your installation')
ImportError: failed to find libmagic.  Check your installation
```

### PR DESCRIPTION
**Pull request by khalibloo**
I want to merge this change because it addresses installation difficulties on windows. It tells poetry to ignore `uwsgi` and also to install `python-magic-bin` for windows.

Closes #4947 

### Pull Request Checklist

1. [ ] Privileged views and APIs are guarded by proper permission checks.
1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Database queries are optimized and the number of queries is constant.
1. [ ] Database migration files are up to date.
1. [ ] The changes are tested.
1. [ ] GraphQL schema and type definitions are up to date.
1. [ ] Changes are mentioned in the changelog.

### CODE REVIEW

**Comment by django-queries** [comment]
Here is the report for e099c259e2a7c89de8051b737fa04f88a5f5daec (khalibloo/saleor @ fix-win-installation)
Base comparison is 5f5936a7d3956ef1fa2bedf61203578901ce8262.



<details><summary>No differences were found. (click me)</summary>
<p>

```diff
# api.benchmark checkout
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  add billing address to checkout            	         34	         34	             20
  add shipping to checkout                   	          7	          7	              0
  checkout payment charge                    	         14	         14	              0
  complete checkout                          	          6	          6	              0
  create checkout                            	         50	         50	             24

# api.benchmark homepage
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  retrieve main menu                         	          5	          5	              0
  retrieve product list                      	          4	          4	              0
  retrieve secondary menu                    	          5	          5	              0
  retrieve shop                              	          2	          2	              0

# api.benchmark product
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  product details                            	         15	         15	              3
  retrieve product attributes                	         13	         13	              2

# api.benchmark variant
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  product variant bulk create                	         51	         51	              3
  retrieve variant list                      	         15	         15	              6

# api product sorting attributes
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  sort product not having attribute data     	         21	         21	              0
```

</p>
</details>

File: pyproject.toml [hunk 1]
```diff
@@ -61,16 +62,16 @@ phonenumberslite = "^8.10.16"
 prices = "^1.0"
 psycopg2-binary = "^2.8.3"
 purl = "^1.5"
+python-magic-bin = "^0.4.14"
```
Comment (maarcingebala) [inline pyproject.toml]: This will install `python-magic-bin` on all platforms, right? Is this what we want or we want to only install it on Windows?

Comment (khalibloo) [reply pyproject.toml]: You're right. I must have missed the marker.

**Comment by NyanKiyoshi** [comment]
Don't forget to update the `requirements*.txt` files!

```
poetry export -f requirements.txt -o requirements.txt --without-hashes
poetry export -f requirements.txt -o requirements_dev.txt --dev --without-hashes
```

**Comment by khalibloo** [comment]
@NyanKiyoshi, thanks.
that's a good candidate for pre-commit. maybe we could also add a check to ensure `poetry.lock` is updated.

File: requirements.txt [hunk 2]
```diff
@@ -49,23 +49,23 @@ docutils==0.15.2
...
 gprof2dot==2016.10.13
-graphene==2.1.8
-graphene-django==2.6.0
+graphene==2.1.6
+graphene-django==2.5.0
 graphene-django-optimizer==0.6.1
 graphene-federation==0.0.3
 graphql-core==2.2.1
-graphql-relay==2.0.0
+graphql-relay==0.4.5
```
Comment (maarcingebala) [inline requirements.txt]: Why are some versions degraded?

Comment (khalibloo) [reply requirements.txt]: I've been looking into this. the only way i could reproduce this from master was by running `poetry update` which is weird in itself. Also, running `poetry lock` gives a different set of versions. Not sure how poetry's dependency resolution works, but it seems inconsistent.

Anyways, I've just reverted the lock file and re-installed the packages. it seems to be ok now.

**Pull request merged**